METRICS.md

```diff
@@ -0,0 +1,17 @@
+# snap collector plugin - use
+
+## Collected Metrics
+This plugin has the ability to gather the following metrics:
+
+
+Namespace | Data Type | Formula | |Description
```

Comment:
What do you think about adding a title to the fourth column, for example, "Range"  or sth more suitable - up to you